File: Candy3/Candy3.py
t=int(input())
for i in range(t):
	s=input()
	n=int(input())
	s=0
	for j in range(n):
		x=int(input())
		s+=x
	if(s%n==0):
		print("YES")
	else:
		print("NO")
	

File: Candy3/Candy3.c
#include<stdio.h>
int main()
{
    long long int t,n,x,i,sum=0;
    scanf("%lld",&t);
    while(t--)
    {
    sum=0;
    scanf("%lld",&n);
    for(i=1;i<=n;i++)
    {
    scanf("%lld",&x);
    x=x%n;
    sum+=x;
    }
    sum%n==0?printf("YES\n"):printf("NO\n");
    }
    return 0;
}
